keygen/src/keygen.rs

```diff
@@ -34,6 +34,8 @@ use std::{
 };
 
 const NO_PASSPHRASE: &str = "";
+// Sentinel value used to write to screen instead of file
+const STDOUT_OUTFILE_TOKEN: &str = "-";
```

Comment:
Would you mind moving this to `clap-utils` and replacing usage there as well? :pray: 

https://github.com/solana-labs/solana/blob/d1db5448b9c3c4f00301163196394e6bb9914a1d/clap-utils/src/keypair.rs#L238